project.clj

```diff
@@ -29,5 +29,6 @@
   :profiles {:dev {:global-vars {*warn-on-reflection* true}
                    :source-paths ["dev"]
                    :resource-paths ["test-resources"]
-                   :dependencies [[org.clojure/tools.namespace "1.3.0"]]}}
+                   :dependencies [[org.clojure/data.json "2.4.0"]
```

Comment:
I would rather prefer not relying on a second JSON library to check the behavior of the one we are using. Would it be possible to check output instead by slurping the file (and optionally decompressing it) and comparing it with a string encoded JSON ?

Reply:
done